admin/css/views/status.css

```
@@ -663,7 +654,237 @@ body.dark-theme .connection-sub-source-off-badge {
+    /* 翻转壳：保留原 connection-item 命中区域，内部做 3D 翻面 */
+    padding: 0;
+    min-height: 140px;
+    cursor: default;
+    perspective: 1400px;
+    /* 外壳本身不再画底色，改由当前可见面承担，避免双层边框 */
+    --card-bg: transparent;
+    --card-border-color: transparent;
+    --card-shadow: none;
+    --card-hover-border-color: transparent;
+    --card-hover-shadow: none;
+    background: transparent;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)